### ISSUE TITLE
Cannot clone to local repo due to "Read-only file system" error, and cannot clone to external repo due to "Not a Git Repository" error.

### ISSUE THREAD

**Issue by WychWitch** [GitJournal/GitJournal#332]
## Environment

**App version:**  1.73.0+2015
**Device information:**
 - OS name and version: Android 9
 - Manufacturer: Samsung
 - Model: Galaxy 8

## Description

**Expected behavior:**
To clone from my gitea installation or gitlab account.

**Current behavior:**
The program fails on the cloning step with a 'Read-only file system' error if the repo is stored internally. If I instead choose to store the repo externally, I get a "Not a Git Repository" error.

Both errors happen regardless if I use a custom host or a gitlab host.

However *creating* a new repo with a gitlab host works for the "Read-only file system" error. It does nothing for the "Not a Git Repository" Error.

## Steps to reproduce

**For reproducing the "Read-only file system error:**
For custom host:
  1. Go through the normal steps to add a custom host

For Gitlab host:
   1. Select automatic set up
   2. Select an already-existing repository
   3. proceed to cloning step

**For reproducing the "Not a Git Repository" Error
Just go through any procedure to create or clone a repo after changing the repo setting to external.

## Others   

<details>

The errors I copy-pasted from the debug section of settings:

```
01:01:46 RepoPath: /data/user/0/io.gitjournal.gitjournal/app_flutter/journal
01:01:46 GitException: failed to create temporary file '/pack_git2_kYboyC': Read-only file system
01:06:02 RepoPath: /data/user/0/io.gitjournal.gitjournal/app_flutter/journal
01:06:02 Not a Git Repository: /data/user/0/io.gitjournal.gitjournal/app_flutter/journal
```

</details>

From the above logs, it looks as though the repo location isn't being changed to the external directory. In the settings menu it says the repo location is set at `/storage/emulated/0/GitJournal/journal` however the error has the repo path at `/data/user/0/io.gitjournal.gitjournal/app_flutter/journal`

**Comment by WychWitch** [comment]
After further testing I discovered that *creating* a new repo works for gitlab, just not cloning an already existing repo. I will update the above to reflect this

**Comment by vHanda** [comment]
Fixed with `1.73.6`, please let me know if you still encounter this issue.

Thanks for reporting it.
